soc/silabs/silabs_siwx91x/Kconfig.defconfig

```diff
@@ -13,6 +13,32 @@ configdefault SYS_CLOCK_TICKS_PER_SEC
 	default 128 if !TICKLESS_KERNEL && SILABS_SLEEPTIMER_TIMER
 	default 1024 if SILABS_SLEEPTIMER_TIMER
 
+config SILABS_COMMANDER_SIGN
+	bool "Sign binary with Silicon Labs Commander"
+	default n
+	select BUILD_OUTPUT_BIN
+	help
+	Enable signing the Zephyr binary with Silicon Labs Commander during build.	
+
+if SILABS_COMMANDER_SIGN
+config M4_OTA_KEY
+	string "M4 OTA Key for Commander"
+	default "Full_OTA_KEY_in_HEX"
+	help
+	The OTA key (hex string) used for signing and encryption with Silicon Labs Commander.
+
+config M4_PRIVATE_KEY
+	string "M4 Private Key Path for Commander"
+	default "Full_path_to_m4_private_key.pem"
+	help
+	Path to the private key file used for signing with Silicon Labs Commander, relative to the board directory.
+
+config COMMANDER_PATH
+	string "Path to Silicon Labs Commander executable"
+	default "Full_path_to_SimplicityCommander-Linux/commander/commander"
+	help
+	Path to the Silicon Labs Commander executable, used for signing the firmware binary.
+endif
```

Comment:
These default do not make much sense IMO, if they are required, leave them as blank and throw an error during the build process.

Also `help` text needs to be indented with an additional 2 spaces after the tab.